base/gfspvgmgr/virtual_group_manager.go

```diff
@@ -646,6 +647,8 @@ type HealthChecker struct {
 	sps          map[uint32]*sptypes.StorageProvider // all sps,  id -> StorageProvider
 	unhealthySPs map[uint32]*sptypes.StorageProvider
 
+	vgm *virtualGroupManager // to fetch self sp id
```

Comment:
A bit complicated.
In refresh loop:
`
for sp in otherSPList:
     checker.addSP(sp)
`
maybe more clear and easy to understand.